src/auth/downscopedclient.ts

```diff
@@ -207,7 +229,7 @@ export class DownscopedClient extends AuthClient {
       subjectTokenType: STS_SUBJECT_TOKEN_TYPE,
     };
 
-    // Exchange the source access token for a Downscoped access token.
+    // Exchange the GCP access token for a Downscoped access token.
```

Comment:
The returned downscoped access token is also a GCP access token. Maybe we can keep this "source AuthClient access token"